plugins/outputs/exec/README.md

```diff
@@ -37,6 +37,9 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
   ## Timeout for command to complete.
   # timeout = "5s"
+  
+  ## Whether the command gets executed once per metric, or once per metric batch
+  # batch_exec = true
```

Comment:
any reason to not use the same config option in other plugins e.g. `use_batch_format`?

Reply:
I was debating that. The reason I went this way is because it's changing the behavior of the plugin, not just how the serialization works. (vs something like the file plugin, which is functionally identical whether use_batch_format is set.) We can just explain that in the readme though and use the same option name.